api-server/package.json

```diff
@@ -1,16 +1,32 @@
 {
   "name": "@freecodecamp/api-server",
-  "private": true,
   "version": "0.0.1",
-  "repository": "freecodecamp/freecodecamp",
+  "description": "The freeCodeCamp.org open-source codebase and curriculum",
+  "license": "BSD-3-Clause",
+  "private": true,
+  "engines": {
+    "node": ">= 14.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/raisedadead/freeCodeCamp.git"
```

Comment:
A quick question - was it intentional to point these to your fork instead of freeCodeCamp/freeCodeCamp?

Reply:
Nope it wasn't. Looks like `npm init` has managed to embarrass me. Lol. Dropping a fix. Thanks for catching.